src/flash.sol

```diff
@@ -77,27 +77,27 @@ contract DssFlash {
 
     // --- Mint ---
     function mint(
-        address _receiver,      // address of conformant IFlashMintReceiver
-        uint256 _amount,        // amount to flash mint [wad]
-        bytes calldata _data    // arbitrary data to pass to the _receiver
+        address bolt,         // Address of conformant IFlashMintReceiver
```

Comment:
I'm torn on this. Aave uses `_receiver`, `_amount`, and `_data` as variable names, so it's easier to grok this module for someone familiar with that pattern (this one is very similar), I'm worried that changing the parameter names here will unnecessarily complicate someone's mental model if they've already got a grasp of the pattern.

Reply:
I agree that since a lot of external users will be interacting with this, it makes sense to get rid of the daiwanese. I do think `dai` and `due` should stay though personally.

Reply:
I'm okay either way. If you guys both want to stick with receiver and amount that makes sense to me. Internally I like the switch to `dai` and `due`, so let's stick with that.

Reply:
Yeah I say we leave receiver, amount, and data and make the other meme changes.

Reply:
Agree is better to leave the conventional names.
Regarding the amount variables, I feel that we should use `wad`/`rad` or otherwise be super explicit `_amount` and `amount45` or `amountRad`.

Reply:
I was thinking that too but `dai` and `due` are both rads

Reply:
Hmm yeah kind of but I just think is better to keep the symmetry on naming between the same amount but in two different units.
Anyway this isn't anything that I would argue strongly at all.